tf2_ros/include/tf2_ros/message_filter.h

```diff
@@ -448,12 +448,9 @@ class MessageFilter : public MessageFilterBase, public message_filters::SimpleFi
   void add(const MConstPtr & message)
   {
     using builtin_interfaces::msg::Time;
```

Comment:
While we are in here, is there any reason for this `using` directive?

Reply:
Is this a better solution?